src/ml/neural_net/quantization_utils.cpp

```diff
@@ -0,0 +1,37 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#include <ml/neural_net/quantization_utils.hpp>
+
+#include <algorithm>
+#include <cmath>
+#include <numeric>
+
+#include <ml/neural_net/float16.h>
+
+namespace turi {
+namespace neural_net {
+
+#ifdef TURI_USE_FLOAT16
```

Comment:
i would do a single check for float16 being enabled across both methods. leave a blank line between the `#if`/`#endif` around function declarations please.

Reply:
Okay let me change it everywhere